x-pack/plugins/apm/server/lib/helpers/get_apm_alerts_client.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { isEmpty } from 'lodash';
-import { APMRouteHandlerResources } from '../typings';
+import { APMRouteHandlerResources } from '../../routes/typings';
```

Comment:
We also have similar client in `apm/server/routes/alerts/alerting_es_client.ts`:

https://github.com/elastic/kibana/blob/4f72a2eb5beded9a3086de7766021d9b65a197f3/x-pack/plugins/apm/server/routes/alerts/alerting_es_client.ts#L14-L35

Should those two clients be co-located? Should they be re-named to better indicate what the difference between them is.

Reply:
Yes, I'll move it to `lib/helpers` as well. There we have: `apm_event_client`, `ml_client` and `internal_saved_objects_client`